### PR TITLE
add ExplicitInvertibleRealTransform

### PR DESCRIPTION
**Pull request by bogovicj**

### CODE REVIEW

File: src/main/java/net/imglib2/realtransform/ExplicitInvertibleRealTransform.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	public ExplicitInvertibleRealTransform( final RealTransform forwardTransform, final RealTransform inverseTransform )
+	{
+		this.forwardTransform = forwardTransform;
+		this.inverseTransform = inverseTransform;
+		this.inverse = new ExplicitInvertibleRealTransform( inverseTransform, forwardTransform, this );
+		// TODO explicitly check that source and target dimensions for forward
+		// and inverse transforms are compatible?
+	}
+
+	private ExplicitInvertibleRealTransform( final RealTransform forwardTransform, final RealTransform inverseTransform, final ExplicitInvertibleRealTransform inverse )
```
Comment (axtimwalde) [inline src/main/java/net/imglib2/realtransform/ExplicitInvertibleRealTransform.java]: How about avoiding possible inconsistency
```java
private ExplicitInvertibleRealTransform( final ExplicitInvertibleRealTransform inverse )
	{
		this.forwardTransform = inverse.inverseTransform;
		this.inverseTransform = inverse.forwardTransform;
		this.inverse = inverse;
	}
```

Comment (axtimwalde) [reply src/main/java/net/imglib2/realtransform/ExplicitInvertibleRealTransform.java]: @bogovicj Can you please do it?  GitHub makes it incredibly hard to contribute to pull requests from private repositories.

Comment (axtimwalde) [reply src/main/java/net/imglib2/realtransform/ExplicitInvertibleRealTransform.java]: @bogovicj forget about it, I made it.  Still hate that they do not make the source clickable...

Comment (bogovicj) [reply src/main/java/net/imglib2/realtransform/ExplicitInvertibleRealTransform.java]: oops, thanks for the reminder, on it

Comment (bogovicj) [reply src/main/java/net/imglib2/realtransform/ExplicitInvertibleRealTransform.java]: I see now, thanks.

**Comment by bogovicj** [comment]
Thanks @axtimwalde

**Pull request merged**